docs/en/contributor/Building-Alluxio-From-Source.md

```diff
@@ -49,8 +49,8 @@ $ mvn -T 2C clean install -DskipTests -Dmaven.javadoc.skip -Dfindbugs.skip \
   -Dcheckstyle.skip -Dlicense.skip -Dskip.protoc
 ```
 
-Note: The flag `-Dskip.protoc` means protoc do not work this time, so it won't generate the
-proto relates source file. So you shouldn't use `-Dskip.protoc` flag at the first build.
+> Note: The flag `-Dskip.protoc` skips generating source files related to gRPC proto.
```

Comment:
Do you think we also have to mention something about ides and the generated-sources directory?

Reply:
Yes!

And I think that fits better to `Contributor-Tools.md`. PTAL at the new changes added.